live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/repository/filter/AspectFilterApplierTest.java

```diff
@@ -79,34 +72,6 @@ void mockBasicData()
         given(mockFilter.aspect()).willReturn(mockAspect);
     }
 
-    @Test
-    void shouldNotFilterOutNonEmptyAspectsWhenEmptyAllowedAndEmptyDenied()
```

Comment:
All removed tests are reflected in `AspectFilterTest` class in common module. 
The ones that remained here are testing the only logic left in tested class which is mapping `null` aspects into empty set.